packages/core/src/html-placeholder.ts

```diff
@@ -55,32 +117,31 @@ export default class Placeholder {
   }
 
   insertScriptTag(src: string) {
-    let newTag = this.end.ownerDocument.createElement('script');
-    for (let { name, value } of [...this.target.attributes]) {
-      if (name === 'type' && value === 'module') {
-        // we always convert modules to scripts
-        continue;
-      }
-      // all other attributes are copied forward unchanged
-      newTag.setAttribute(name, value);
-    }
-    newTag.src = src;
+    let newTag = makeTag(this.end.ownerDocument, { from: this.target, attributes: { src } });
+    normalizeScriptTag(newTag);
+
```

Comment:
It seems like we do need both the changes in `ember-html.ts` as well as in this file. (@ef4 was pondering if we needed both in the other PR). I tried removing these changes, but the new tests I added started failing on the `<link>` tags, even though the `<script>` tags were fine. I did not investigate further.